app/views/oauth_clients/index.html.erb

```diff
@@ -15,7 +15,7 @@
   </thead>
   <% @tokens.each do |token| %>
     <tr>
-      <td><%= link_to token.client_application.name, token.client_application.url %></td>
+      <td><%= link_to token.client_application.name, token.client_application.url, target: "_blank" %></td>
```

Comment:
We rarely use `target: "_blank"` for links - any particular reason to add it to this one?

Also this line causes the CI failures for both hash rocket syntax, and noopener when using target: "_blank"

Reply:
I would argue that any link that does not remain in the current application should be opened in a new tab. This makes users less likely to lose their way if they click on the OAuth application name. There are many more external links which should maybe also be opened in a new tab. It would be even better to annotate them with an icon like below to notify users of such links.

![image](https://user-images.githubusercontent.com/1073881/97108386-8cb96880-16cd-11eb-9e3d-5b3ea7c940c3.png)